docs/changelog-configuration.md

```diff
@@ -255,6 +255,13 @@ The default is `["rst"]`.
 A list of output formats to write the changelog as. Supported formats
 are `rst` for ReStructuredText and `md` for MarkDown.
 
+### `changelog_nice_yaml`(boolean)`
```

Comment:
```suggestion
### `changelog_nice_yaml` (boolean)
```

Also, can you move this option up so that the options are again alphabetically ordered? (I'll move `output_formats` in another PR, that one is also wrongly placed.)